deploy/olm-catalog/common-service-operator/0.0.3/common-service-operator.v0.0.3.clusterserviceversion.yaml

```diff
@@ -93,7 +93,7 @@ metadata:
     createdAt: "2020-01-06T10:16:16Z"
     description: The Common Service Operator provides a Kubernetes CRD-Based API to
       manage the lifecycle of IBM cloud common services.
-    repository: https://github.com/IBM/common-service-operator
+    repository: https://github.com/IBM/meta-operator
     support: IBM
   name: common-service-operator.v0.0.3
```

Comment:
meta-operator?

Reply:
How about change the file name to meta-operator as well?

Reply:
let's refactor one by one.

we need to create new repository and application on quay.io for this new name.